### ISSUE TITLE
Minimalist Version?

### ISSUE THREAD

**Issue by JacobDB** [VPenkov/okayNav#38]
This looks awesome, but it's a bit more full featured than I'd like. What I'd really like is if it could just move elements in and out of the container, without any styles, click events, or animations built in. Any chance you'll put out okayNav Light any time soon?

**Comment by cognopatrick** [comment]
If you don't use the css and dont use transitions and swipe events you just could make a light version yourself?

**Comment by JacobDB** [comment]
True, but I think the project performs well, and it seems like a waste to rebuild something more basic when this already exists. I think it'd be relatively easy for the maintainer of this project to strip all the fancy stuff out. I'll wait and see what they think before moving forward with my own project.
